litestar/typing.py

```diff
@@ -318,12 +318,12 @@ def is_required(self) -> bool:
     @property
     def is_annotated(self) -> bool:
         """Check if the field type is Annotated."""
-        return Annotated in self.type_wrappers  # type: ignore[comparison-overlap]
+        return bool(self.metadata)
```

Comment:
This is a non-related change - so I can split out into a separate PR if anyone feels necessary - it just seems better to infer this property this way, as the only way the object can have metadata is if it is annotated, and a type cannot be annotated without at least one metadata argument.

Reply:
I don't think there's any need to split this out into separate PR.